src/loaders/svg/tvgSvgPath.cpp

```diff
@@ -55,12 +55,22 @@ static bool _parseNumber(char** content, float* number)
 static bool _parseFlag(char** content, int* number)
 {
     char* end = NULL;
-    *number = strtol(*content, &end, 10);
-    //If the start of string is not number or a number was a float
-    if ((*content) == end || *end == '.') return false;
-    //If a flag has a different value than 0 or 1
-    if (*number != 0 && *number != 1) return false;
-    *content = _skipComma(end);
+    //NOTE: It handles the case what the flag of the arc of the path is defined
```

Comment:
Itself is NOTE. do we really need to emphasize "I'm a note ?"

Suggest format:

/* It handles the case what the flag of the arc of the path is defined
    without spaces or commas, such as 00, 01, 11 or 10. */

Reply:
I updated patch